docs/guide/extending_online_editor.md

```diff
@@ -255,3 +255,16 @@ In the example above, the template files for the front end could be:
 ```
 
 Templates for Content View in the Back Office would be `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_word.html.twig` and `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_block.html.twig` respectively (assuming Admin SiteAccess uses the `admin` theme).
+
+## Plugins configuration
+
+To enable new CKEditor plugins in Alloy Editor you need to define them in RichText AlloyEditor Semantic Configuration.
```

Comment:
@dew326 your input here would be valuable. Is it about these https://ckeditor.com/cke4/addons/plugins/all? If so, do we just need to pass them based on that list or are additional actions required. It would be useful to enrich our doc with such precise information.

Reply:
Definitely agree. We need to be clear here.

Reply:
Yes, you need to develop or download a plugin to the CKEditor. You need to add them to the CKEditor plugins by `add` method. If you download from the CKEditor you probably need to only include them on a page.
Example of our plugin: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-add-content.js#L98

Reply:
Activate a plugin is one thing, make it work with our storage engine is a different story.
@alongosz I think we should be clear on potential limitations, especially with DocBook.